src/main/java/io/vertx/ext/asyncsql/package-info.java

```diff
@@ -185,7 +185,9 @@
  *   "password" : <your-password>,
  *   "database" : <name-of-your-database>,
  *   "charset" : <name-of-the-character-set>,
- *   "queryTimeout" : <timeout-in-milliseconds>
+ *   "queryTimeout" : <timeout-in-milliseconds>,
+ *   "sslmode" : <"disable"|"prefer"|"require"|"verify-ca"|"verify-full">,
+ *   "sslrootcert" : <path to file with certificate>
```

Comment:
`sslMode` and `sslRootCert` would be more appropriate names (`maxPoolSize` and `queryTimeout` are camel cased as well)